### PR TITLE
SDK - XSIAM items into validate

### PR DESCRIPTION
**Pull request by abaumgarten**
## Status
- [] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://github.com/demisto/etc/issues/48151)
related: https://github.com/demisto/content/pull/18740

## Description
Added new XSIAM items into the sdk-validate. For now, all validations will pass with no checking. 

## Screenshots
image

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,7 @@
 * Fixed an issue where concurrent processes created by **lint** caused deadlocks when opening files.
 * Fixed an issue in the **format** command where `_dev` or `_copy` suffixes weren't removed from the subscript names in playbooks and layouts.
 * Fixed an issue where **validate** failed on nonexistent `README.md` files.
+* Added the XSIAM content items into the **validate** command.
```
Comment (dorschw) [inline CHANGELOG.md:13]:
```suggestion
* Added support of XSIAM content items to the **validate** command.
```

File: demisto_sdk/commands/common/hook_validations/correlation_rule.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+class CorrelationRuleValidator(ContentEntityValidator):
+    """
+    CorrelationRuleValidator is designed to validate the correctness of the file structure we enter to content repo.
+    """
+
+    def is_valid_file(self, validate_rn=True, is_new_file=False, use_git=False):
+        """
+        Check whether the correlation rule is valid or not
+        """
+        return True
```
Comment (dorschw) [inline demisto_sdk/commands/common/hook_validations/correlation_rule.py:18]: Please mention [this](https://github.com/demisto/etc/issues/48151#issuecomment-1109660727) link in the code, explain that for now we return True regardless of the item content, and add a logger warning/debug message stating this (`automatically considering XSIAM content item as valid, see issue #48151`)

please add this mention in all XSIAM validators

File: demisto_sdk/commands/validate/validate_manager.py [hunk 3]
```diff
@@ -1606,7 +1677,8 @@ def check_file_relevance_and_format_path(self, file_path, old_path, old_format_f
         if file_type in [FileType.PYTHON_FILE, FileType.POWERSHELL_FILE, FileType.JAVASCRIPT_FILE, FileType.XIF_FILE]:
             if not (str(file_path).endswith('_test.py') or str(file_path).endswith('.Tests.ps1') or
                     str(file_path).endswith('_test.js')):
-                file_path = file_path.replace('.py', '.yml').replace('.ps1', '.yml').replace('.js', '.yml').replace('.xif', '.yml')
+                file_path = file_path.replace('.py', '.yml').replace('.ps1', '.yml').replace('.js', '.yml').replace(
+                    '.xif', '.yml')
```
Comment (dorschw) [inline demisto_sdk/commands/validate/validate_manager.py:1681]: Please use a `Path` object:
```python
if path.suffix != '.py': 
    path=path.with_suffix('.yml')
```

Comment (abaumgarten) [reply demisto_sdk/commands/validate/validate_manager.py:1681]: not part of my changes.

File: demisto_sdk/commands/common/hook_validations/correlation_rule.py (hunk 2 above)
Comment (abaumgarten) [reply demisto_sdk/commands/common/hook_validations/correlation_rule.py:18]: Done

**Review by dorschw — APPROVED** [review]

**Pull request merged**